pandas/tests/test_graphics.py

```diff
@@ -1065,12 +1065,6 @@ def test_implicit_label(self):
         self._check_text_labels(ax.xaxis.get_label(), 'a')
 
     @slow
-    def test_explicit_label(self):
```

Comment:
Sorry! I didn't realize the test was for the *axis* label. Good call on removing it.